include/mbedtls/pk.h

```diff
@@ -447,30 +409,22 @@ int mbedtls_pk_can_do_psa(const mbedtls_pk_context *pk, psa_algorithm_t alg,
  *                    \p usage, exporting and copying the key, and
  *                    possibly other permissions as documented for the
  *                    \p usage parameter.
- *                    The permitted algorithm policy is determined as follows
- *                    based on the #mbedtls_pk_sigalg_t type of \p pk,
- *                    the chosen \p usage and other factors:
- *                      - #MBEDTLS_PK_SIGALG_RSA_PKCS1V15 whose underlying
- *                        context uses the PKCS#1 v1.5 padding mode:
+ *                    For keys created with \c mbedtls_pk_wrap_psa(), the
+ *                    permitted algorithm is the same as the original PSA key.
```

Comment:
> the permitted algorithm is the same as the original PSA key.

The thing is, a PSA key does not in general have “the permitted algorithm”. It could be the same permitted algorithm _policy_ (including the enrollment algorithm), or it could be the PSA key's primary algorithm (which is what it was in 3.6, and AFAIK this hasn't changed).